src/dparse/lexer.d

```diff
@@ -497,13 +498,14 @@ public struct DLexer
      *         other token text.
      *     haveSSE42 = Parse streaming SIMD Extensions 4.2 in inline assembly
      */
-    this(ubyte[] range, const LexerConfig config, StringCache* cache,
+    this(R)(R range, const LexerConfig config, StringCache* cache,
         bool haveSSE42 = sse42()) pure nothrow @safe
+    if (is(Unqual!(ElementEncodingType!R) : ubyte) && isDynamicArray!R)
     {
         this.haveSSE42 = haveSSE42;
         auto r = (range.length >= 3 && range[0] == 0xef && range[1] == 0xbb && range[2] == 0xbf)
             ? range[3 .. $] : range;
-        this.range = LexerRange(r);
+        this.range = LexerRange(cast(const(ubyte)[]) r);
```

Comment:
You can use `representation` from `std.string` which does the same (though that's just a style nit)

Reply:
Because `std.string` is not imported i won't change this.